### DISCUSSION TITLE
FSK communication with SX1276 problem

### DISCUSSION THREAD

**Discussion by yrnloc** [jgromes/RadioLib#462] [General]
Hi, 
I use two SX1276 (https://www.amazon.de/gp/product/B084NWDFH1) on two arduino pro mini.  
The lora communication works fine, but I am not able to receive something I send using FSK.
I can receive messages from homematic devices using the commented out code, but nothing that I send myself.

Is there something I did wrong?
[SX127x_Receive_Interrupt.ino.txt](https://github.com/jgromes/RadioLib/files/8008079/SX127x_Receive_Interrupt.ino.txt)
[SX127x_Transmit_Interrupt.ino.txt](https://github.com/jgromes/RadioLib/files/8008080/SX127x_Transmit_Interrupt.ino.txt)

**Comment by jgromes** [comment]
It is very unclear to me which parts of the code you are actually using. Please post a minimum example to reproduce the issue (without commented-out code).

That being said, I did notice your transmitter and receiver are configured differently:

Transmitter is at 24 kHz frequency deviation:
```c++
int state = radio.beginFSK(868.29, 10, 24, 100, 10, 4, false);
```

While receiver is at 20 kHz:
```c++
int state = radio.beginFSK(868.29, 10, 20, 100, 10, 4, false);
```

**Reply by yrnloc** [reply]
Thank you for your answer. Here is the minimum  example to reproduce the issue. I used 24 kHz frequency 
deviation only for testing. Now transmitter and receiver is at 20kHz.
[SX127x_Receive_Interrupt.ino.txt](https://github.com/jgromes/RadioLib/files/8053669/SX127x_Receive_Interrupt.ino.txt)
[SX127x_Transmit_Interrupt.ino.txt](https://github.com/jgromes/RadioLib/files/8053672/SX127x_Transmit_Interrupt.ino.txt)
